cypress-custom/integration/fee.test.ts

```diff
@@ -54,9 +54,53 @@ describe('Fetch and persist fee', () => {
 
   // TODO: not sure if it's easy to test this
   it('Re-fetched when it expires', () => {
+    const NETWORK = ChainId.RINKEBY.toString()
+    const TOKEN = 'ETH'
+    const KEY = 'redux_localstorage_simple_fee'
```

Comment:
refactor this with the foreach, and move up? sth like`const FEE_LOCAL_STORAGE_KEY` ?